holoviews/plotting/widgets/__init__.py

```diff
@@ -338,13 +358,12 @@ def get_widgets(self):
                 init_dim_vals.append(dim_vals[0])
                 dim_vals = escape_list(escape_vals(dim_vals))
                 next_vals = escape_dict({k: escape_vals(v) for k, v in next_vals.items()})
-
             visibility = '' if visible else 'display: none'
             dim_str = dim.pprint_label
             escaped_dim = dimension_sanitizer(dim_str)
             widget_data = dict(dim=escaped_dim, dim_label=dim_str,
                                dim_idx=idx, vals=dim_vals, type=widget_type,
-                               visibility=visibility, step=step, next_dim=next_dim,
+                               visibility=visibility, step=step, next_dim=next_dim or None,
```

Comment:
``or None`` is an unrelated bug fix?

Reply:
Not unrelated no, to do with templating the variable now that it is a function argument rather than templated directly into the JS code.

Reply:
On a more general note, there is probably a relatively clean path to progressively moving to the widget system we envision. I'd probably start by moving away from templating and encapsulate all this stuff in a JSON blob.